README.md

```diff
@@ -17,7 +17,7 @@ The transformation:
 - expects it to have a specified field;
 - expects the value of the field to be either `INT64` or `org.apache.kafka.connect.data.Timestamp` and not be `null`.
 
-Exists in one variant:
+Exists in two variants:
 - `io.aiven.kafka.connect.transforms.ExtractTimestamp$Value` - works on values.
```

Comment:
Missing the second variant

Reply:
So far I removed it. Will prepare as a separate PR

Reply:
True